docs/a320-simvars.md

```diff
@@ -1019,6 +1019,33 @@
         - 2
         - 3
 
+- A32NX_PAX_TOTAL_ROWS_{rows}
+    - Number
+    - Indicates the current number of pax in the selected rows
+    - {rows}
+        - 1_6
+        - 7_13
+        - 14_21
+        - 22_29
+
+- A32NX_PAX_TOTAL_ROWS_{rows}_DESIRED
+    - Number
+    - Indicates the target number of pax in the selected rows
+    - {rows}
+        - 1_6
+        - 7_13
+        - 14_21
+        - 22_29
+
+- PAYLOAD STATION WEIGHT:{stationIndex}
```

Comment:
Why is this documented if it's an AVar and part of MSFS ?

Reply:
Added it mainly to keep track of payload station indices, it can be omitted if not required